# มุมข้อต่อทั้ง 4 (หน่วยเป็นองศา หรือ เรเดียน แล้วแต่ตกลง)
float64 j1
float64 j2
float64 j3
float64 j4

# สถานะกริปเปอร์ (0=ปล่อย, 1=จับ)
int32 gripper_state

# ปุ่ม Safety (True=ขยับได้, False=หยุดฉุกเฉิน)
bool is_enabled
